server/fixtures/user.js

```diff
@@ -1,9 +1,26 @@
-const newUser = {
+export const newUser = {
   email: 'lewislulu@yahoo.com',
   username: 'lulu',
   firstName: 'Lewis',
   lastName: 'Thomas',
   bio: 'Welcome to my world.',
   password: '123456'
 };
-export default newUser;
+export const goodUserData = {
+  email: 'testing@authorshaven.com',
+  username: 'testing',
+  lastName: 'testing',
+  firstName: 'testing',
+  password: 'testing'
+};
+export const emptyUserData = {
+  email: '',
+  username: '',
+  lastName: '',
+  firstName: '',
+  password: ''
+};
+export const badUserData = args => ({
```

Comment:
This is a function, I expect it to be named like on. Maybe `getUSerData`

Reply:
Done.